### ISSUE TITLE
[Visual Requirements - Change Visualization->Library Panels]: Luminosity ratio is less than 4.5:1 for “Data source: Elasticsearch” link present under 'Library panel’ blade.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66309]
### User Experience:  
Low vision users would find it difficult to read text, if a minimum luminosity contrast ratio of 4.5:1 is not provided between text and its background. Note: User credentials should NOT be included in the bug.

### Repro Steps:
 Open #Url: https://play.grafana.org/  Sign in with valid credentials.  "Home" page appears.  Navigate the Feature Showcase pane heading and select it. Navigate the expanded Dropdown and select the "Edit" menu item.  Navigate the page "Edit" page which appears. Navigate to the "Change Visualization" button and select it.  Navigate to the "Library Panels" tab which appears and select. Navigate the Library panel. Run CCA Tool Verify the issue.

### Actual Result: 
Issue: Luminosity ratio for “Data source:Elasticsearch” link is 1.4:1 ratio which is less than minimum required ratio 4.5:1 present under 'Library panel’ blade.

### Expected Result:
Solution: Luminosity ratio should bemore than or equal to 4.5:1 for Data source: Elasticsearch” link presentunder 'Library panel’ blade.
Note:  Same issue exists for hhhhhh... and product status text under Library panel blade  Luminosity ratio of placeholder text is less than 4.5:1 throughout the application.

### Severity
`SEV2`

**Comment by khushijain21** [comment]
I couldn't reproduce this issue, can you direct exactly how?

**Comment by kaydelaney** [comment]
Reiterating @khushijain21 's comment. Also not sure where the "Data source: Elasticsearch" link mentioned is

**Comment by natellium** [comment]
@kaydelaney I'm not sure either. But looking at that list could be the contrast in the disabled items?

**Comment by natellium** [comment]
Fixed by https://github.com/grafana/grafana/pull/69907
